queue_services/common/src/entity_queue_common/service_utils/service_logger.py

```diff
@@ -41,14 +45,11 @@ def before_breadcrumb(crumb, hint):  # pylint: disable=unused-argument; callback
     )
 
 logging.basicConfig(
-    level=logging.INFO,
+    level=LOG_LEVEL,
     format='%(asctime)s,%(msecs)d-%(name)s-%(levelname)s > %(module)s:%(filename)s:%(lineno)d-%(funcName)s:%(message)s',
     datefmt='%H:%M:%S',
 )
-logging.getLogger('asyncio').setLevel(logging.DEBUG)
-
-if os.getenv('DISABLE_HTTP_ACCESS_LOGS', 'False').lower() == 'true':
-    # This disables the healthz / readyz / meta logging messsages.
-    logging.getLogger('aiohttp.access').setLevel(logging.ERROR)
+logging.getLogger('asyncio').setLevel(LOG_LEVEL)
+logging.getLogger('aiohttp.access').setLevel(LOG_LEVEL)
```

Comment:
The variable `DISABLE_HTTP_ACCESS_LOGS` doesn't appear to be used.  By setting the log level as an environment variable we can't set different log levels, but we have more control to reduce the logging volume.

Reply:
you might want if to check if `DISABLE_HTTP_ACCESS_LOGS` really isn't being used anywhere.  I think you added this env var @seeker25?